client/packages/system/src/Asset/api/hooks/document/useAssets.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery, useUrlQueryParams } from '@openmsupply-client/common';
+import { useAssetApi } from '../utils/useAssetApi';
+
+export const useAssets = () => {
+  const { queryParams } = useUrlQueryParams({
```

Comment:
Does code need to be added here?
I can't seem to search by code..

![image](https://github.com/msupply-foundation/open-msupply/assets/8287373/cb1c992e-526a-44b7-8a33-9b98a7408b36)